conference.js

```diff
@@ -411,28 +315,6 @@ class ConferenceConnector {
             room.leave(CONFERENCE_LEAVE_REASONS.UNRECOVERABLE_ERROR).then(() => connection.disconnect());
```

Comment:
Since we never assign anything to the `connection` I believe this would fail.

Reply:
We do assign it, link above ... that has not changed. Probably when we move and the conference/room handling in react we will drop it, but that will be the last thing from conference.js.
Well not sure how we will drop everything because of jitsi-meet-torture but something we can decide later :)